interface/forms/dictation/report.php

```diff
@@ -18,8 +39,8 @@ function dictation_report( $pid, $encounter, $cols, $id) {
         $value = "yes";
       }
       $key=ucwords(str_replace("_"," ",$key));
-      print "<td><span class=bold>" . xlt($key) . ": </span><span class=text>" .
-        nl2br(text($value)) . "</span></td>";
+      print "<td><span class='bold'>" . xlt($key) . ": </span><span class='text'>" .
+        nl2br($value) . "</span></td>";
```

Comment:
But leaving it there causes the output to show the html characters.  I have changed the order and it still outputs the line breaks to the screen.